mediator/med.F90

```diff
@@ -2009,8 +2015,23 @@ subroutine DataInitialize(gcomp, rc)
             if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
             ! Create mesh info data
-            call med_meshinfo_create(is_local%wrap%FBImp(n1,n1), &
-                 is_local%wrap%mesh_info(n1), is_local%wrap%FBArea(n1), rc=rc)
+            call ESMF_FieldBundleGet(is_local%wrap%FBImp(n1,n1), fieldCount=fieldCount, rc=rc) 
+            if (ChkErr(rc,__LINE__,u_FILE_u)) return
+
+            if (fieldCount == 0) then           
+              if (mastertask) then
```

Comment:
I am wondering if we could use ESMF_LogWrite to replace writing log file just for mastertask? It might be easy to follow if all the prints are in the same file.

Reply:
Of course we could do it but I tried to follow the convention as much as possible and these kind of logs goes to mediator log file in general

Reply:
@junwang-noaa Yes. Actually there is no strict rule about it but in general the mediator related information that is read from the ESMF config file goes to the PET files such as list of exchange fields, their connection information with other components, grid information, log about which subroutine is called and also if debug level is increased the min and max values of the exchange fields etc.  Also, the log related with the unsupported configurations such as setting coupling mode that is not supported currently goes to PET files. The mediator log file is generally used to show the activated components and their interaction (as a matrix), the list of exchange fields and information about the mapping and merging and created route handles etc. Since that part that you commented is implemented recently to allow one-way coupling with the activated component, I put this informative message to mediator log because PET files have too much information and user could easily skip this message but mediator log file is more clean way to show it. Of course we could always make modification, if you really want to see them in the PET files.

Reply:
It is good to me. It's OK to keep the code as is.